build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -138,7 +147,9 @@
     torch_models.EFFICIENTNET_B7_FP32_TORCH,
 ]
 
-CUDA_MODELS_LONG = RESNET50_TF_BATCHES + BERT_LARGE_TF_BATCHES + T5_LARGE_TF_BATCHES + BERT_LARGE_TORCH_BATCHES + RESNET50_TORCH_BATCHES
+CUDA_MODELS_LONG = RESNET50_TF_BATCHES + BERT_LARGE_TF_BATCHES + T5_LARGE_TF_BATCHES + \
+    BERT_LARGE_TORCH_BATCHES + RESNET50_TORCH_BATCHES + \
+    RESNET50_FP16_TORCH_BATCHES
```

Comment:
We try to follow Google Python style, which discourages to use `\` to wrap lines. (https://google.github.io/styleguide/pyguide.html#32-line-length)
```suggestion
CUDA_MODELS_LONG = (RESNET50_TF_BATCHES + BERT_LARGE_TF_BATCHES + T5_LARGE_TF_BATCHES +
    BERT_LARGE_TORCH_BATCHES + RESNET50_TORCH_BATCHES +
    RESNET50_FP16_TORCH_BATCHES)
```